### PR TITLE
feat(StoneDB 8.0): tianmu_insert_delayed decides the binary log file events. (#723)

### LINKED ISSUE #723: feature(StoneDB 8.0): add some mtr test case
**Is your feature request related to a problem? Please describe.**

port some test case from 5.7 to 8.0:
    ./suite/tianmu/t/alter_table_v1.test
    ./suite/tianmu/t/issue364..test 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by lujiashun**
## Summary about this PR

Issue Number: close #723

[summary]
 1 tianmu_insert_delayed=1,insert DML binary log will be convert to "load into outfile ..." with other insert DMLs;
 2 tianmu_insert_delayed=0,insert DML binary log is the normal binary log events;

## Tests Check List

- [ ] Unit test
- [X] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [x] New Feature
- [ ] Bug Fix
- [ ] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

**Review by DandreChen — APPROVED** [review]
LGTM.

File: sql/sql_insert.cc [hunk 1]
```diff
@@ -681,7 +681,8 @@ bool Sql_cmd_insert_values::execute_inner(THD *thd) {
 
     if (!has_error ||
         thd->get_transaction()->cannot_safely_rollback(Transaction_ctx::STMT)) {
-      if (mysql_bin_log.is_open()) {
+      bool tianmu_engine = insert_table->s->db_type() ? insert_table->s->db_type()->db_type == DB_TYPE_TIANMU: false;
+      if (mysql_bin_log.is_open()&& !tianmu_engine) {
```
Comment (RingsC) [inline sql/sql_insert.cc]: I think that should be like this: do tianmu check firstly, then binlog flag check. 
if (!tianmu_engine && mysql_bin_log.is_open())

**Review by RingsC — CHANGES_REQUESTED** [review]

**Review by RingsC — APPROVED** [review]

**Review by DandreChen — APPROVED** [review]
LGTM.

**Review by RingsC — APPROVED** [review]
LGTM

**Review by hustjieke — APPROVED** [review]
LGTM

**Pull request merged**